### PR TITLE
S3 integration test

### PR DESCRIPTION
**Pull request by jasiu001**
Related to [#5102](https://github.com/kyma-project/kyma/issues/5102)

- Add S3 integration test based on minio
- remove GCS addon provider

### CODE REVIEW

**Comment by polskikiel** [comment]
After review:
- added tmp dir name randomizer
- added docs about integration test prerequisites

File: README.md [hunk 1]
```diff
@@ -126,7 +126,16 @@ To set up the project, download these tools:
 
 Before each commit, use the `before-commit.sh` script which tests your changes and builds binaries. If you want to run the Helm Broker locally, read [this](/docs/run-local.md) document.
 
-Integration tests check if all parts of the Helm Broker work together. They start kube-apiserver with etcd, set up the Controller and the Broker. Install [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) before you run integration tests. Run tests using the following command:
+Integration tests check if all parts of the Helm Broker work together.
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
You can also run integration tests that check if all parts of the Helm Broker work together. 
```

File: README.md [hunk 2]
```diff
@@ -126,7 +126,16 @@ To set up the project, download these tools:
 
 Before each commit, use the `before-commit.sh` script which tests your changes and builds binaries. If you want to run the Helm Broker locally, read [this](/docs/run-local.md) document.
 
-Integration tests check if all parts of the Helm Broker work together. They start kube-apiserver with etcd, set up the Controller and the Broker. Install [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) before you run integration tests. Run tests using the following command:
+Integration tests check if all parts of the Helm Broker work together. 
+
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -126,7 +126,16 @@ To set up the project, download these tools:
...
+These are the prerequisites for integration tests:
+
+- [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder)
+- [Etcd](https://github.com/etcd-io/etcd/releases/tag/v3.4.0)
+- [Minio](https://min.io/download)
+
+Run tests using the following command:
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
Run integration tests using this command:
```

File: README.md [hunk 4]
```diff
@@ -126,7 +126,16 @@ To set up the project, download these tools:
 
 Before each commit, use the `before-commit.sh` script which tests your changes and builds binaries. If you want to run the Helm Broker locally, read [this](/docs/run-local.md) document.
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
Before each commit, use the `before-commit.sh` script. The script runs unit tests that check your changes and build binaries. If you want to run the Helm Broker locally, read [this](/docs/run-local.md) document.
```

**Pull request merged**